api/src/neo4j/queries/search.js

```diff
@@ -327,13 +327,13 @@ const _search = async ({
   // the EC field for reaction could contain ":", which is a special character
   // in this case th search term is modified to be escape and perform an exact match
   const term = searchTerm.includes('EC:')
-    ? `\\"${searchTerm}~\\"`
-    : `${searchTerm}~`;
+    ? `\\"${searchTerm}\\"`
+    : `${searchTerm}`;
 
   // Metabolites are not included as it would mess with the limit and
   // relevant metabolites should be matched through CompartmentalizedMetabolites
   let statement = `
-CALL db.index.fulltext.queryNodes("fulltext", "${term}")
+CALL db.index.fulltext.queryNodes("fulltext", "${term} ${term}~")
```

Comment:
A comment would be nice here, to avoid future confusion

Reply:
Great point @MalinAhlberg ! Added a comment about this now.